fern/pages/sdks/introduction/configuration.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'SDK Configuration'
+subtitle: 'Configure your SDK generators in `generators.yml` and publish to registries'
+---
+
+<Note>
+  Before you can configure your generators, you must setup your API Definition. Learn more [**here**](learn/api-definition/introduction/what-is-the-fern-folder)
+</Note>
+
+## Top Level Configuration
+
+```yaml title="generators.yml" maxLines=10
```

Comment:
Any reason to not show a little bit more? Maybe 19 lines to show a full example?

Reply:
Chatted offline -- planning to make this an accordion with: 

- SDK published to GitHub
- SDK published to private registry